src/tests/ftest/launch.py

```diff
@@ -514,8 +514,11 @@ def check_remote_output(task, command):
     # Create a dictionary of hosts for each unique return code
     results = dict(task.iter_retcodes())
 
+    # Create a list of any hosts that timed out
+    timed_out = [str(hosts) for hosts in task.iter_keys_timeout()]
```

Comment:
Just a side comment that creating a copy of of a list, but of a different type, can be done with:
```suggestion
    timed_out = list(map(str, task.iter_keys_timeout()))
```
In this case, it really doesn't matter. But `map` returns an object that can be iterated (lazily) so you can do this for large lists/mappings:
```
for thing in map(type, some_list):
    # do it
```